src/io/internal/pipelines/dicom/CMakeLists.txt

```diff
@@ -10,18 +10,31 @@ find_package(ITK REQUIRED
   )
 include(${ITK_USE_FILE})
 
+set(cpp_nlohmann_json_GIT_REPOSITORY "https://github.com/nlohmann/json.git")
+set(cpp_nlohmann_json_GIT_TAG  bc889afb4c5bf1c0d8ee29ef35eaaf4c8bef8a5d)
+FetchContent_Declare(
+  cpp_nlohmann_json
```

Comment:
I need to remove this now. Its no longer needed, as we have decided to use rapidjson.